R/framework_ebp.R

```diff
@@ -45,9 +48,19 @@ framework_ebp <- function(fixed, pop_data, pop_domains, smp_data, smp_domains,
                                     levels = levels_tmp)
   pop_domains_vec <- pop_data[[pop_domains]]
 
-
   smp_data[[smp_domains]] <- factor(smp_data[[smp_domains]],
                                     levels = levels_tmp)
+
+  if(is.null(aggregate_to) != TRUE){
```

Comment:
is.null should be tested in if statement, does not need to be compared with boolean. Also, spaces should be considered. if () { etc.